src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/AssemblyBasedCallerUtils.java

```diff
@@ -829,9 +852,18 @@ private static VariantContext phaseVC(final VariantContext vc, final String ID,
         final List<Genotype> phasedGenotypes = new ArrayList<>();
         for ( final Genotype g : vc.getGenotypes() ) {
             final List<Allele> alleles = g.getAlleles();
-            if (phaseGT.equals(phase10) && g.isHet()) Collections.reverse(alleles); // swap the alleles if heterozygous
+            final List<Allele> newAlleles = new ArrayList<>(alleles);
+            if (phaseGT.equals(phase10) && g.isHet()) {
```

Comment:
This is truly petty but I would prefer checking `g.isHet()` first, since the phase genotype string is only relevant if it's a het.

Reply:
done